cmd/controller-manager/app/options/options.go

```diff
@@ -79,4 +83,6 @@ func (o *Options) AddFlags(flags *pflag.FlagSet) {
 	flags.DurationVar(&o.ClusterStatusUpdateFrequency.Duration, "cluster-status-update-frequency", 10*time.Second,
 		"Specifies how often karmada-controller-manager posts cluster status to karmada-apiserver.")
 	flags.BoolVar(&o.LeaderElection.LeaderElect, "leader-elect", true, "Start a leader election client and gain leadership before executing the main loop. Enable this when running replicated components for high availability.")
+	flags.StringVar(&o.WatchedKinds, "watchedKinds", "", "Indicates which API objects are meaningful to karmada-controller.")
```

Comment:
Multi resource are split by ",", right? You can give an example here, such as "pod, service, deployment"

Reply:
yes, it's comma-delimited